z/rtutil.go

```diff
@@ -40,21 +40,23 @@ type stringStruct struct {
 }
 
 //go:noescape
-//go:linkname aeshash runtime.aeshash
-func aeshash(p unsafe.Pointer, h, s uintptr) uintptr
+//go:linkname memhash runtime.memhash
+func memhash(p unsafe.Pointer, h, s uintptr) uintptr
 
-// AESHash is the hash function used by map, it utilizes available hardware instructions.
+// MemHash is the hash function used by map, it utilizes available hardware instructions(behaves as
```

Comment:
`go maps` instead of `map` would make more sense.

Reply:
Done.